README.md

```diff
@@ -156,25 +156,31 @@ Below is an example using ERB:
 <% end %>
 
 <%= link_to "Sign out", destroy_user_session_path, :method => :delete %>
-
 ```
 
-#### Enable TOTP support for existing users
+#### Upgrading from version 1.X to 2.X
+
+The following database fields are new in version 2.
+
+- `direct_otp`
+- `direct_otp_sent_at`
+- `totp_timestamp`
+
+To add them, generate a migration such as:
+
+    $ rails g migration AddTwoFactorFieldsToUsers direct_otp:string direct_otp_sent_at:datetime totp_timestamp:timestamp
 
-If you have existing users that need to be provided with a OTP secret key, so
-they can use TOTP, create a rake task. It could look like this one below:
+The `otp_secret_key` is not only required for users who use Google Authentictor,
```

Comment:
Maybe instead of naming only Google Authenticator, say something like "authenticator apps such as Google Authenticator, Authy, etc.."?